common/lib/xmodule/xmodule/course_module.py

```diff
@@ -652,6 +663,31 @@ class CourseFields(object):
             {"display_name": _("None"), "value": CATALOG_VISIBILITY_NONE}]
     )
 
+    entrance_exam_enabled = Boolean(
+        display_name=_("Entrance Exam Enabled"),
+        help=_(
+            "Specify whether students must complete an entrance exam before they can view your course content." +
+            "Note, you must enable Entrance Exams for this course setting to take effect."),
+        default=False,
+        scope=Scope.settings,
+    )
+
+    entrance_exam_minimum_score_pct = Float(
+        display_name=_("Entrance Exam Minimum Score (%)"),
+        help=_(
+            "Specify a minimum percentage score for an entrance exam before students can view your course content." +
+            "Note, you must enable Entrance Exams for this course setting to take effect."),
+        default=65,
+        scope=Scope.settings,
+    )
+
+    entrance_exam_id = String(
+        display_name=_("Entrance Exam ID"),
+        help=_("Content module identifier (location) of entrance exam."),
+        default=None,
+        scope=Scope.settings,
+    )
+
```

Comment:
re: export/import, gotcha. In the meantime, I don't know if we should a) document this limitation or b) somehow explicitly error out (aka assert) so that we don't get authors into a bad state. I'd prefer a hard assertion rather than silent failures, but it wouldn't be a must have.